Makefile

```diff
@@ -1000,6 +1000,10 @@ RELEASE_TAG ?= $(shell git describe --abbrev=0 2>/dev/null)
 ifneq (,$(findstring -,$(RELEASE_TAG)))
     PRE_RELEASE=true
 endif
+## Check if previous tag is provided, if not set empty
+ifeq (,$(findstring -,$(PREVIOUS_VERSION_TAG)))
```

Comment:
Isn't PREVIOUS_VERSION_TAG already empty if it's not provided?

Reply:
yaa, variable `PREVIOUS_VERSION_TAG` was not at all getting set but It can be done other way around also, let me update it.